moveit_setup_assistant/src/widgets/setup_assistant_widget.h

```diff
@@ -220,6 +221,7 @@ private Q_SLOTS:
   EndEffectorsWidget* efw_;
   VirtualJointsWidget* vjw_;
   PassiveJointsWidget* pjw_;
+  ControllersWidget* cw_;
```

Comment:
these abbreviated variable names are bad practice, but they are my fault 7 years ago when I was a younger programmer. can you add a TODO to make them not abbreviated? I don't want to make this PR any bigger itself.